packages/devnet/package.json

```diff
@@ -3,42 +3,18 @@
     "version": "2.0.0-alpha.3",
     "license": "Apache-2.0",
     "scripts": {
-        "build": "run-s compile export deploy:anvil",
-        "clean:ignored": "rimraf artifacts build cache coverage deployments/localhost dist",
-        "clean:hardhat": "hardhat clean",
-        "clean": "run-s clean:hardhat clean:ignored",
-        "compile": "hardhat compile",
-        "deploy": "hardhat deploy",
-        "deploy:anvil": "hardhat --network localhost deploy-anvil --dump-file build/anvil_state.json --silent",
-        "deploy:localhost": "hardhat deploy --network localhost --export export/abi/localhost.json --reset",
-        "dev": "hardhat node --export export/abi/localhost.json",
-        "export": "hardhat export-artifacts export/artifacts --hide-sources --including-empty-bytecode --including-no-public-functions",
-        "prepack": "run-s build tsc",
-        "tsc": "tsc"
+        "build": "run-s build:forge build:cannon",
+        "build:forge": "forge soldeer install && forge build",
+        "build:cannon": "cannon build --write-deployments deployments --anvil.dump-state anvil_state.json",
+        "clean": "cannon clean",
+        "dev": "cannon run"
```

Comment:
The `cannon run` command expects package references to be passed as arguments.
Do you intend to run `pnpm dev [packageRefs...]`?

Reply:
No, I'll remove the script